usage-insights-launch.mdx

```diff
@@ -0,0 +1,89 @@
+![Pro workspace overview](images/usage-metrics-launch/pro-usage-overview.png)
+
+[Free plan](/docs/concepts/pricing#free-plan) workspaces can also get a glimpse into their usage on the workspace landing page. This concisely shows the percentage of available [free tier resources](/docs/concepts/pricing#pricing-plans) that have been used in the current calendar month.
+![Free workspace usage overview](images/usage-metrics-launch/free-usage-overview.png)
+
+## Technical details
+
+As you would expect, the data we track comes from a variety of sources and we wanted to collect it all into a single place.
+
+In order to power this feature we had to choose a database that is well suited to collect and report time series data and also met our usage requirements:
+
+- Fast inserts
+- Fast and flexible querying (ideally using SQL)
+- Support for rollup and eviction of old date
+- Efficient on disk storage
+- Support for high cardinality metrics
+
+After investigating and experimenting with many different databases we eventually landed on [Clickhouse](https://clickhouse.com/) because it meets all of the above requirements on paper and after some real world experimentation they all held up in practice.
+
+### Architecture
+
+In order to collect the data in (near) real time, we have two collection strategies:
+
+1.  Poll usage data from existing backends (Postgres, Search, Attachments)
+2.  React to real time usage events (AI questions, branch changes)
+
+**Polling**
+
+We have an existing task scheduler and background job processor which we use to periodically spawn collection jobs in each region. These jobs then concurrently fan out to all databases within their region, collecting usage data, collating it and then storing it in Clickhouse using their [bulk insert](https://clickhouse.com/docs/en/cloud/bestpractices/bulk-inserts) mechanism.
+
+**Event processing**
+
+For events caused by user actions we rely on the fact we store metadata about these events in DynamoDB. AWS provides a number of mechanisms for reacting to changes in Dynamo data and in our case we send change events to [Kinesis](https://aws.amazon.com/pm/kinesis/?trk=76c15db0-b03d-4489-8fe0-42aff7b6e211&sc_channel=ps&ef_id=Cj0KCQjwncWvBhD_ARIsAEb2HW9zGa3r1jh-nzDKgk5C1rMZRK7bdwnIRJRxh3hdsW48wzVwj2ooGr8aArRlEALw_wcB:G:s&s_kwcid=AL!4422!3!651612449030!e!!g!!amazon%20kinesis!19828211163!146688868243&gclid=Cj0KCQjwncWvBhD_ARIsAEb2HW9zGa3r1jh-nzDKgk5C1rMZRK7bdwnIRJRxh3hdsW48wzVwj2ooGr8aArRlEALw_wcB) which in turn spawns a lambda function which writes the event into Clickhouse. Although there are a more moving parts, it allows us to record these events almost instantaneously as well as providing a level of resilience in the face of failures since events can build up in the queue until Clickhouse is ready to receive them.
+
+**On disk compression**
```

Comment:
@ryanslade you had mentioned wanting to add a screenshot around compression. Anything interesting we can show here?

Reply:
Done, PTAL.